src/mcg.rs

```diff
@@ -42,6 +42,14 @@ enum OscSource {
     External = 2,
 }
 
+pub enum CpuFreq {
```

Comment:
I'm uncertain on how much I like the values of this enum, but it does well enough for now.